eqs/src/eth_polling.rs

```diff
@@ -289,10 +290,12 @@ impl EthPolling {
                 }
                 CAPEEvents::FaucetInitializedFilter(filter_data) => {
                     let ro_bytes = filter_data.ro_bytes;
+                    let pub_key: UserPubKey =
+                        bincode::deserialize(&filter_data.faucet_manager_pub_key.to_vec()).unwrap();
```

Comment:
This won't work with the deployment script which uses UTF-8 bytes from the TaggedBase64 serilization.